### DISCUSSION TITLE
editing only a single layer

### DISCUSSION THREAD

**Discussion by shoeb8997** [geoman-io/leaflet-geoman#844] [Questions & Help]
My usecase is if there are multiple layer plotted on the map.I want to enable editing mode of only one layer based on the layerid.Means only one layer will be in editing mode.

**Comment by Falke-Design** [comment]
You can enable the edit mode for the layer with `layer.pm.enable()` and disable it with `layer.pm.disable()`

So to enable a layer by id call `map._layers[YOUR_ID].pm.enable()`
